fs_testing_pipe.py

```diff
@@ -0,0 +1,20 @@
+import dlt
```

Comment:
keep those in some ignored folder ;)

Reply:
yeah, i have that also, but since i am working on two different machines i need to do this sometimes ;)